cmd/cache-server/main.go

```diff
@@ -36,7 +36,10 @@ import (
 func main() {
 	rootDir := flag.String("root-directory", ".kcp-cache", "Path to the root directory where all files required by this server will be stored")
 
-	var cacheServerFlags, remainingFlags []string
+	var (
```

Comment:
IMHO pre-allocating here does nothing for performance (unless someone passes dozens of args ...) and makes the readability worse. Not sure we "need" this

Reply:
I agree there is some places where this is less effective, I can revert instances like above if we prefer and just add a //nolint comment; was striving for consistency at first

Reply:
I would probably nolint these?

Reply:
Done, thanks!